activesupport/test/concern_test.rb

```diff
@@ -128,4 +128,12 @@ def test_raise_on_multiple_included_calls
       end
     end
   end
+
+  def test_no_raise_on_same_included_call
+    assert_nothing_raised do
+      2.times do
+        load File.expand_path("../fixtures/some_module.rb", __FILE__)
```

Comment:
I think you can do `require_relative` here, right?

Reply:
Using `require_relative` rather than `load` would result in a no-op on the second iteration, no?